src/Compilers/VisualBasic/Portable/Lowering/StateMachineRewriter/StateMachineRewriter.vb

```diff
@@ -375,6 +379,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
         End Sub
 
+        Friend Function EnsureSpecialMember(member As SpecialMember, bag As DiagnosticBag) As Symbol
+            Return Binder.GetSpecialTypeMember(F.Compilation.Assembly, member, Me.Body.Syntax, bag)
+        End Function
+
+        ''' <summary>
+        ''' Check that the property and its getter exist and collect any use-site errors.
+        ''' </summary>
+        Friend Sub EnsureSpecialPropertyGetter(member As SpecialMember, bag As DiagnosticBag)
+            Dim symbol = DirectCast(EnsureSpecialMember(member, bag), PropertySymbol)
+
+            If symbol IsNot Nothing Then
+                Dim getter = symbol.GetMethod
+
+                If getter Is Nothing Then
+                    Binder.ReportDiagnostic(Diagnostics, Body.Syntax, ERRID.ERR_NoGetProperty1, CustomSymbolDisplayFormatter.QualifiedName(symbol))
```

Comment:
There is already `Diagnostics` in scope. The purpose here is to capture diagnostics in a separate bag, so I picked a distinctive name.